framework/web/UrlManager.php

```diff
@@ -117,14 +117,20 @@ class UrlManager extends Component
      *
      * After the UrlManager object is created, if you want to change this property,
      * you should only assign it with a cache object.
-     * Set this property to false if you do not want to cache the URL rules.
+     * Set this property to `false` if you do not want to cache the URL rules.
      */
     public $cache = 'cache';
     /**
      * @var array the default configuration of URL rules. Individual rule configurations
      * specified via [[rules]] will take precedence when the same property of the rule is configured.
      */
     public $ruleConfig = ['class' => 'yii\web\UrlRule'];
+    /**
+     * @var UrlNormalizer|array|boolean the configuration for [[UrlNormalizer]] used by this UrlManager.
+     * If `false` normalization will be skipped. If `true`, normalizer will use default settings.
+     * @since 2.0.10
+     */
+    public $normalizer = true;
```

Comment:
If this property was named `normalize`, setting it to `true` would make sense. As it contains the `normalizer` object true is not really fitting here. 

Having a difference between the options for UrlRule and for UrlManager is no problem because UrlManager has a default config, while UrlRule has a default fallback to what is configured in UrlManager.

My main point here is consistency with other places where similar object dependencies exist.

For UrlRule it should be null like e.g. `yii\web\User::$accessChecker`: https://github.com/yiisoft/yii2/blob/09b1fd47e071acdc9ec0c1988684d367fb067e02/framework/web/User.php#L107-L112

- null: use the default instance from UrlManager
- false: disable it (similar to Sort and Pagination in DataProviders)
- array: a config to create custom normalizer.

For UrlManager:

- `['class' => 'yii\web\UrlNormalizer']` - default value
- false to disable
- array for other config.